src/components/Filters/Filters-container.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid/Grid';
+
+import useFiltersStyles from './Filters-styles';
+
+const Filters = () => {
+  const classes = useFiltersStyles();
+  const blockchains: string[] = ['BTC', 'ETH', 'XRP', 'LTC', 'BCH', 'XTZ', 'ADA', 'EOS', 'XLM'];
```

Comment:
Czy te skróty walutowe będą jeszcze gdzieś potrzebne? Jeśli tak, to może warto je wynieść do jakiegoś oddzielnego pliku, w którym ogólnie są/będą consty? To tylko propozycja ;)

Reply:
Wydaje mi się, że na ten moment nigdzie indziej tego nie będziemy potrzebować. Zobaczę jak przejdę do połączenia filtrów. Wtedy to przeniose